packages/frontend/src/components/Explorer/ResultsCard/QuickCalculations.tsx

```diff
@@ -90,13 +93,36 @@ const getSqlForQuickCalculation = (
     quickCalculation: QuickCalculation,
     fieldReference: string,
     sorts: SortField[],
+    warehouseType: WarehouseTypes | undefined,
 ) => {
+    let fieldQuoteChar = '"';
+    if (warehouseType) {
+        switch (warehouseType) {
+            case WarehouseTypes.BIGQUERY:
+            case WarehouseTypes.DATABRICKS:
+                fieldQuoteChar = '`';
```

Comment:
personal opinion: when dealing with this type of logic, I prefer a function-const + switch + return, rather than a `let variable` + switch + break.

Reply:
sounds good, will change 👍

Reply:
done: https://github.com/lightdash/lightdash/pull/9704/commits/96e540daad1a3e3e633ae5c96684b171e5d6a85d